docs/operations/cpu_hotplug.md

```diff
@@ -38,23 +38,31 @@ spec:
     - LiveMigrate
 ```
 
-### Enable in VM spec
-The VM object should look like this:
+### Configure the VM rollout strategy
+Hotplug requires a VM rollout strategy of `LiveUpdate`, so that the changes made to the VM object propagate to the VMI without a restart.
+This is also done in the KubeVirt CR configuration:
+
 ```yaml
 apiVersion: kubevirt.io/v1
-kind: VirtualMachine
+kind: KubeVirt
 spec:
-  liveUpdateFeatures:
-    cpu: {}
+  configuration:
+    vmRolloutStrategy: "LiveUpdate"
 ```
-Specifying the `cpu.maxSockets` value is optional. If you leave it unset, it will default to 4 times the `sockets` value.
 
-**NOTE:** In order for these changes to take effect the VM needs to be rebooted.
+More information can be found on the [VM Rollout Strategies](./vm_rollout_strategies.md) page
+
+### [OPTIONAL] Set maximum sockets or hotplug ratio
+You can explicitly set the maximum amount of sockets in three ways:
+
+1. with a value VM level
+2. with a value at the cluster level
+3, with a ratio at the cluster level (`maxSockets = ratio * sockets`).
```

Comment:
Typo: The comma after '3' is preventing that line from being displayed as a numbered item.